src/pages/[userId]/prospects.js

```diff
@@ -47,45 +54,6 @@ export default function Prospects() {
     }
   `;
 
-  const prospects = [
-    {
-      id: 1,
-      names: ['Jon smith'],
-      preferred_name: null,
-      status: 'contacted',
-      last_contacted: '07/13',
-      notes: ['Meeting up on 08/27', 'Left a voicemail', 'Emailed'],
-      other_ministers: ['Reagann Smith', 'Ryan Bristow'],
-    },
-    {
-      id: 2,
-      names: ['Matt Clark'],
-      preferred_name: null,
-      status: 'pledged, no amount',
-      last_contacted: '06/21',
-      notes: ['Need to follow up'],
-      other_ministers: ['Danielle Clark'],
-    },
-    {
-      id: 3,
-      names: ['Mitchell Pavel'],
-      preferred_name: null,
-      status: 'contacted',
-      last_contacted: '06/30',
-      notes: ['Wants to donate, partnering with the wife'],
-      other_ministers: [],
-    },
-    {
-      id: 4,
-      names: ['Bob Chin'],
-      preferred_name: null,
-      status: 'No answer',
-      last_contacted: '07/01',
-      notes: ['Left voicemail', 'Attempted contact; No response.'],
-      other_ministers: [],
-    },
-  ];
-
   return (
     <Container>
       <h1>Prospects for {userId}</h1>
```

Comment:
We actually need to get the minister by id here and use their name instead of just the id.

Reply:
Since you'll be making another get request, you'll want to make sure you use good variable names for `data`, isLoading`, and `error` on each.

Reply:
Done. Let me know if it needs to be changed again